electron-app.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/main/package.json b/main/package.json
+index c4fd488e30644e247e86f14a0ec12e82d3cf27c1..92d238c4b9b4d639b446c0e7936aa2e2ae456944 100644
+--- a/main/package.json
++++ b/main/package.json
+@@ -4,7 +4,7 @@
+   "description": "Visual Programming Language for Exploring Math, Music, and Programming",
+   "main": "main.js",
+   "scripts": {
+-    "start": "electron .",
++    "start": "electron . --no-sandbox",
```

Comment:
You should use zypak instead of disabling the sandbox, see other electron apps on Flathub for an example.

Reply:
Hi, Thank you so much for your help! I was originally having problems running electron because of an issue with the configuration of the SUID sandbox helper binary and was told to just not use the sandbox (because flatpak already has one) as a fix. Will zypak fix the original configuration problem?

Reply:
I'm also not entirely sure how to implement zypak... I'm a little lost looking through the examples that use it. The `run.sh` files that they create have a bunch more commands that I don't understand and it looks pretty different from the `npm start` used to run the app in the Flatpak [electron-sample-app](https://github.com/flathub/electron-sample-app) example I based the app on.

I've also pasted below the console output of what I got when I tried to run the app in the first place without `--no-sandbox`. I did check the file that it referenced and the permissions seem to be correct so perhaps it's a symptom of a different issue (which was fixed when I added `--no-sandbox`. According to some basic Googling it seems like this is a relatively common electron issue with AppImage and Flatpak, but if you know any more about it, any help would be greatly appreciated!

```
The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without
sandboxing I'm aborting now. You need to make sure that
/app/share/musicblocks/node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.
```

Thanks so much for your help.
@bilelmoussaoui

Reply:
If there's no `chrome-sandbox` binary in the same directory as your Electron app, do something like

```bash
echo 'true' > chrome-sandbox
chmod +x chrome-sandbox
```

which should make it work. This is a deficiency of sorts where Zypak expects the a chrome-sandbox to exist that can be overridden (which is something I plan on changing, not hard but haven't gotten the chance).